pkg/config/analysis/local/istiod_analyze.go

```diff
@@ -376,28 +377,30 @@ func (sa *IstiodAnalyzer) AddRunningKubeSourceWithRevision(c kubelib.Client, rev
 	// TODO: are either of these string constants intended to vary?
 	// We gets Istio CRD resources with a specific revision.
 	krs := sa.kubeResources.Remove(kuberesource.DefaultExcludedSchemas().All()...)
-	if remote {
-		krs = krs.Remove(kuberesource.DefaultRemoteClusterExcludedSchemas().All()...)
-	}
-	store := crdclient.NewForSchemas(c, crdclient.Option{
-		Revision:     revision,
-		DomainSuffix: "cluster.local",
-		Identifier:   "analysis-controller",
-		FiltersByGVK: map[config.GroupVersionKind]kubetypes.Filter{
-			gvk.ConfigMap: {
-				Namespace:    sa.istioNamespace.String(),
-				ObjectFilter: kubetypes.NewStaticObjectFilter(isIstioConfigMap),
+
+	// FIXME: Multi-cluster istio c r d is not analyzed for now
+	if !remote {
+		store := crdclient.NewForSchemas(c, crdclient.Option{
+			Revision:     revision,
+			DomainSuffix: "cluster.local",
+			Identifier:   "analysis-controller",
+			FiltersByGVK: map[config.GroupVersionKind]kubetypes.Filter{
+				gvk.ConfigMap: {
+					Namespace:    sa.istioNamespace.String(),
+					ObjectFilter: kubetypes.NewStaticObjectFilter(isIstioConfigMap),
+				},
 			},
-		},
-	}, krs)
-	sa.stores = append(sa.stores, store)
+		}, krs)
+		sa.stores = append(sa.stores, store)
+	}
 
 	// We gets service discovery resources without a specific revision.
 	krs = sa.kubeResources.Intersect(kuberesource.DefaultExcludedSchemas())
 	if remote {
-		krs = krs.Remove(kuberesource.DefaultRemoteClusterExcludedSchemas().All()...)
+		multiAnalyzer := &multicluster.ServiceAnalyzer{}
```

Comment:
Can we use this one? 

https://github.com/istio/istio/blob/881ad80653fc38299dd49e6bee519b34dfe9c7e5/pkg/config/analysis/analyzers/all.go#L81-L85

Then if we add more multicluster analyzers, we won’t need to handle them here.

Reply:
Is great idea